fenix/app/src/androidTest/assets/pages/generic3.html

```diff
@@ -10,6 +10,7 @@ <h1>
     </p>
     <p>
         <a href="../resources/washington.pdf">PDF file</a>
+        <a href="../resources/pdfForm.pdf">PDF form file</a>
```

Comment:
@AndiAJ @sv-ohorvath 
I'd prefer if we can to consolidate our asset use, as ultimately this inflates the size of the APK.

In looking at the other PDF file, I see that it's roughly ~300kb, and the form is much smaller.

Can we replace and delete the `washington.pdf` file with your form PDF? If so, can we rename it to `dummy.pdf` and use it in the tests we use the former `washington.pdf`

If not, can we remove `Washington.pdf` and use this dummy PDF instead? 
https://www.w3.org/WAI/ER/tests/xhtml/testfiles/resources/pdf/dummy.pdf

Reply:
Merged the 2 resources into 1 [pdfForm.pdf](https://github.com/mozilla-mobile/firefox-android/pull/2424/files#diff-dca97f3aed817ef86fdb7b88f732e3bb72b4b6bdf8910bf756c2d366ac109bb2)

The dummy PDF isn't suitable for our tests, we need to have plain text, hyperlinked text, hyperlinked image, and fillable fields.

Reply:
Fair enough, I think we can compress the PDF outside of this ticket to at least shrink it down a bit.